pydictable/core.py

```diff
@@ -100,12 +100,16 @@ def __get_field_by_type_hint(cls, type_hint):
     @classmethod
     def get_fields(cls) -> Dict[str, Field]:
         fields = {}
-        for attr in inspect.getmembers(cls):
-            if isinstance(attr[1], Field):
-                fields[attr[0]] = attr[1]
+        dictable_fields = {k: v for k, v in dict(vars(cls)).items() if isinstance(v, Field)}
         for name, th in get_type_hints(cls).items():
-            if name not in fields:
+            if name in dictable_fields:
+                fields[name] = dictable_fields[name]
+            else:
                 fields[name] = cls.__get_field_by_type_hint(th)
```

Comment:
Depending on get_type_hints() method for the order, since all fields will have type hints.